### DISCUSSION TITLE
Date formatting

### DISCUSSION THREAD

**Discussion by maticboncina** [firefly-iii/firefly-iii#5225] [Firefly III]
Hello,

Is there a way to change the formatting from mm/dd/yyyy to dd/mm/yyyy? I only found one guide, but that was for the Portugal language pack. 

How could I do that on the English one? I don't mind which is used in terms of if it is GB/US/AU...

Thank you in advance :)

**Comment by JC5** [comment]
It follows your browsers language.

**Reply by maticboncina** [reply]
Hm, okay. I tried switching between English US, English UK and it remained the same.

**Reply by JC5** [reply]
Wait, are you talking about the input fields or how it is displayed? 😁

**Reply by maticboncina** [reply]
Input fields. It was actually the only way how I checked. I assume that input fields have the same formatting as actual display fields? 😅

**Reply by JC5** [reply]
No, input fields are formatted by the browser and it depends greatly on the browser how they are shaped. The language setting of your browser should influence it.
